src/m365/tenant/commands/service/service-report-historicalservicestatus.ts

```diff
@@ -0,0 +1,112 @@
+import auth from '../../../../Auth';
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import Command, { CommandOption } from '../../../../Command';
+import Utils from '../../../../Utils';
+
```

Comment:
Hi @waldekmastykarz  - Is there a way how you pick the _empty lines_ and _dangling spaces_? Do you use some tool or do your **eyes** do that for you 😮? If it is former, then it is good news for me, if it is latter, then I may to increase **my attention to detail** level? 🤔
Just curious so that next time I can spare your time further during the review. 😊

Reply:
Empty lines I catch myself. Call it months of experience ;)
Dangling spaces are highlighted in VSCode by markdownlint.

Reply:
Then the only option remains is to "mock" (_mocha terminology_) your eyes. lol. 🤨.

Thanks for the lead about markdownlint. Will start using that and hopefully your review effort would be minimized in the next PRs. Thanks again for introducing new learning items. 🙌

Reply:
I appreciate you taking so much effort into getting your PRs just right. It makes a difference. Your work is superb 👏

Reply:
Thanks a lot for the kind words.. Super happy to hear that right from the horse's mouth.. 🕺..

M365 CLI Maintainers is already doing heavy work by reviewing all the PRs. Just trying my part to minimize your job so that you can continue built super awesome things and inspiring the community... ✌✌

Reply:
Thank you 😊